processor/transformprocessor/internal/metrics/metrics_test.go

```diff
@@ -1622,3 +1622,7 @@ func createNewTelemetry() (pmetric.ExemplarSlice, pcommon.Map, pcommon.Value, pc
 func strp(s string) *string {
 	return &s
 }
+
+func intp(i int64) *int64 {
```

Comment:
Is this used?

Reply:
Ya its used in `metrics/functions_test.go`. It is defined in the `traces_test` and `logs_test` file as well despite being used different places.